ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+    - chain: storage-conf-csi-optional-gcp-filestore
```

Comment:
If the csi chain is placed behind `cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision`, how `cucushift-installer-check-gcp-user-labels` can run the csi verification?

Reply:
Hmm, wait, I remember there are some csi checking in `cucushift-installer-check-gcp-user-labels` before, but now they are removed?

Reply:
I guess you are talking about the steps `cucushift-installer-check-gcp-user-labels-pd-csi` and `cucushift-installer-check-gcp-user-labels-filestore-csi`, right? Yes, I removed them after discussing with @Phaow who said they will include the tests in E2E tests instead. Thanks!

Reply:
Good to know. @Phaow if that is the case, shall we remove the `storage-conf-csi-optional-gcp-filestore` chain together here, if the e2e testing is ready to cover it, we can add the chain back.

Reply:
Yeah, I added the check in e2e phase, I'll try to finish https://github.com/openshift/openshift-tests-private/pull/12494 ASAP. But the `storage-conf-csi-optional-gcp-filestore chain` is still needed as check with @jianli-wei , the case need to check both pd and filestore volumes tags.

Reply:
Got it.